packages/core/helper-plugin/lib/src/providers/CustomFieldsProvider/index.js

```diff
@@ -0,0 +1,31 @@
+/**
+ *
+ * CustomFieldsProvider
+ *
+ */
+import React from 'react';
+import PropTypes from 'prop-types';
+import CustomFieldsContext from '../../contexts/CustomFieldsContext';
+
+const CustomFieldsProvider = ({ children, customFields }) => {
+  return (
+    <CustomFieldsContext.Provider
+      value={{
```

Comment:
I'm wondering why we don't just provide the customFields instance directly?

Reply:
I thought it would be a better experience when consuming the useCustomFields hook, but it's true that it doesn't really make a difference. I'll update it